packages/app/src/components/Header/index.tsx

```diff
@@ -192,8 +207,9 @@ const Header = ({ data }: { data: any }) => {
               <Button
                 color="inherit"
                 onClick={(e) => { handleAttributeClick(e) }}
-                aria-label="Open attributes selection menu"
+                aria-label="Attributes selection menu"
                 aria-haspopup="menu"
+                tabIndex={4 + ALTTEXT_SIDEBAR_TABS}
```

Comment:
Technically that could work, but I'm not sure it's useful because the tab order isn't always reflected by the code execution order. If we wanted to change the tab order of items in the header, we'd have to undo that change. Similarly, I'm not sure a function would be useful because it would need to know the desired tab order of the element it's being called in.

Reply:
Yeah I do see how that could have some issues based on code execution order.